spk/sickbeard-custom/src/installer.sh

```diff
@@ -88,19 +92,21 @@ preuninst ()
 {
     # Stop the package
     ${SSS} stop > /dev/null
-
+    
     # Remove the user if uninstalling
-    if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ]; then
-        syno_group_remove
+    
+	if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ] && [ "${BUILDNUMBER}" -lt "7135" ]; then
+    	syno_group_remove
 
         delgroup ${USER} ${GROUP}
         deluser ${USER}
-    fi
+  
 
-    # Remove firewall config
-    if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ]; then
-        ${SERVICETOOL} --remove-configure-file --package ${PACKAGE}.sc >> /dev/null
-    fi
+    	# Remove firewall config
+    	if [ "${SYNOPKG_PKG_STATUS}" == "UNINSTALL" ]; then
```

Comment:
If you're already checking for PKG_STATUS on L98, then this check is no longer needed.